docs/source/playbooks.rst

```diff
@@ -133,21 +131,21 @@ and group variables files may help you organize your variable values more
 easily. Included with the sample playbook is a sample variables
 file `all.yml`_.
 
-The value for property **BPXK_AUTOCVT** must be configured to ``ON``.
+The value for the property **BPXK_AUTOCVT** must be configured to ``ON``.
 
-The value for property **ZOAU_HOME** is the ZOA Utilities install root path,
+The value for the property **ZOAU_HOME** is the ZOA Utilities install root path;
 for example, ``/usr/lpp/IBM/zoautil``.
 
-The value for property **PYTHONPATH** is the ZOA Utilities Python library path,
-for example, ``/usr/lpp/IBM/zoautil/lib/``.
+The value for the property **PYTHONPATH** is the ZOA Utilities Python library
+path; for example, ``/usr/lpp/IBM/zoautil/lib/``.
 
-The value for property **LIBPATH** is both the path to the python libraries on
-the target and the ZOA Utilities Python library path separated by
-semi-colons ``:``, for example,
+The value for the property **LIBPATH** is both the path to the Python libraries
+on the target and the ZOA Utilities Python library path separated by
+colons ``:``; for example,
 ``/usr/lpp/IBM/zoautil/lib/:/usr/lpp/rsusr/python36/lib:/lib:/usr/lib:.``.
 
-The value for property **PATH** is the ZOA utilities BIN path and Python
-interpreter path, for example,
+The value for the property **PATH** is the ZOA utilities BIN path and the Python
+interpreter path; for example,
 ``/usr/lpp/IBM/zoautil/bin;/usr/lpp/rsusr/python36/bin/python``.
```

Comment:
@RadhaV2020 pleas fix this, i know i have fixed it more than once :( somehow it keeps coming back and is a pain point for customers, remove the semi-colon ";" and replace with colon ":". and add `bin`
``/usr/lpp/IBM/zoautil/bin;/usr/lpp/rsusr/python36/bin/python`` 
TO 
``/usr/lpp/IBM/zoautil/bin:/usr/lpp/rsusr/python36/bin/python:/bin``

Reply:
ack. will update